package.json

```diff
@@ -13,6 +13,7 @@
   "workspaces": {
     "packages": [
       "apps/**/*",
+      "configs/**/*",
```

Comment:
Would it make sense to have this package at `/packages/configs` (similar to how we organise `packages/tools`)?

Reply:
With the exception of `pie-icons-configs`, everything in `packages/*` is useful for customers and I believe we should have a way of segregating customer oriented packages that are published from internal/tooling/dev oriented code that doesn't need to be published.